src/Interfaces/ConfigInterface.php

```diff
@@ -7,10 +7,10 @@ interface ConfigInterface
     /**
      * Create a new instance of a ConfigInterface object.
      *
-     * @param array|string $context Raw array of configuration options or path to a
+     * @param null $context Raw array of configuration options or path to a
```

Comment:
This is an interface which specifies the expected types of the concrete implementation.

```suggestion
     * @param array|string $context Raw array of configuration options or path to a
```